src/Domain/WikiContentRendererFactory.php

```diff
@@ -11,27 +11,17 @@
 class WikiContentRendererFactory implements ContentRendererFactory {
 
 	public function createContentRenderer( RendererConfig $config ): ContentRenderer {
-		if ( $config->language !== '' ) {
+		if ( $config->render ) {
+			return new MarkdownRenderer();
+		}
+		else {
```

Comment:
No need for the `else` block. You can just put the `return` code below the `if` block.